src/Configuration/Config.KeyPerFile/src/KeyPerFileConfigurationBuilderExtensions.cs

```diff
@@ -19,6 +19,18 @@ public static class KeyPerFileConfigurationBuilderExtensions
         /// <param name="optional">Whether the directory is optional.</param>
         /// <returns>The <see cref="IConfigurationBuilder"/>.</returns>
         public static IConfigurationBuilder AddKeyPerFile(this IConfigurationBuilder builder, string directoryPath, bool optional)
```

Comment:
Should I also add another overload with only directoryPath as input, with optional and reloadOnChange set to false to match other providers?

Reply:
Yes